ios/ImageCropPicker.m

```diff
@@ -429,6 +429,25 @@ - (NSDictionary*) createAttachmentResponse:(NSString*)filePath withExif:(NSDicti
              };
 }
 
+// See https://stackoverflow.com/questions/4147311/finding-image-type-from-nsdata-or-uiimage
+- (NSString *)determineMimeTypeFromImageData:(NSData *)data {
```

Comment:
I am aware of this comment: https://github.com/ivpusic/react-native-image-crop-picker/pull/389/files#r129528647

I'd still suggest this solution for performance reasons as it seems to be very reliable mechanism.

Reply:
@fubar Hello, forgive if I am asking silly question here, if we have to implement this, what changes we have to do in our code. I have implemented the library and using single image capture as well as multiple image capture functionality and it is working great. Can you tell me the procedure to skip the image compression in iOS.

Reply:
This will be available in the next release.